src/id3manager/formats/toml.py

```diff
@@ -26,6 +26,14 @@ def read(self, fileobj: t.IO) -> t.List[id3.Frame]:
                     frame = frame_cls(text=[frame["text"]])
                 elif issubclass(frame_cls, id3.UrlFrame):
                     frame = frame_cls(url=frame["url"])
+                elif issubclass(frame_cls, id3.APIC):
+                    mime = frame.get("mime")
+                    description = frame.get("description")
+                    type = getattr(id3.PictureType, frame.get("type", "COVER_FRONT"))
+
+                    frame = utils.parse_apic(frame["data"], mime, type)
```

Comment:
So TOML, in a way, becomes superior to plain text because it allows to to specify mimetype and apic type. If possible, I'd prefer us to remain compatible and express this via plain/text too.

Can we come up with some sort of custom formatting for the value?

```
APIC = image/png;https://example.com/logo.png;COVER_FRONT
```

I'm not sure that `;` is a good delimiter though. We probably should pick one that is _rarely_ used in URLs. :thinking:

Reply:
Right. TOML is definitely more expressive, and plain text would restrict our options here... But I see your point.

What if we keep the MIME type only for "data" URLs that allow embedding images (https://developer.mozilla.org/en-US/docs/Web/URI/Schemes/data#syntax), but completely ignore it for `file://` and `http://` style URLs? That would leave us with two values - the URL and the image type. We could then make the latter optional (with the default value  `COVER_FRONT`), and separate them by ` `?